lightning/src/ln/channel.rs

```diff
@@ -2804,22 +2803,22 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			return Err(ChannelError::Close("Peer attempted to reestablish channel with a very old local commitment transaction".to_owned()));
 		};
 
-		// We increment cur_remote_commitment_transaction_number only upon receipt of
+		// We increment cur_counterparty_commitment_transaction_number only upon receipt of
 		// revoke_and_ack, not on sending commitment_signed, so we add one if have
 		// AwaitingRemoteRevoke set, which indicates we sent a commitment_signed but haven't gotten
 		// the corresponding revoke_and_ack back yet.
-		let our_next_remote_commitment_number = INITIAL_COMMITMENT_NUMBER - self.cur_remote_commitment_transaction_number + if (self.channel_state & ChannelState::AwaitingRemoteRevoke as u32) != 0 { 1 } else { 0 };
+		let next_remote_commitment_number = INITIAL_COMMITMENT_NUMBER - self.cur_counterparty_commitment_transaction_number + if (self.channel_state & ChannelState::AwaitingRemoteRevoke as u32) != 0 { 1 } else { 0 };
```

Comment:
I guess this where I'm a bit confused. Here we are mixing "counterparty" and "remote".

Reply:
My bad, I think this variable was really badly-named. Switched it to `next_counterparty_commitment_number` as we used this value to build counterparty's commitment transaction.